bgpd/rfapi/bgp_rfapi_cfg.c

```diff
@@ -1389,7 +1389,7 @@ DEFUN (vnc_export_mode,
 		vty_out(vty,
 			"Changing modes for zebra export not implemented yet\n");
 		return CMD_WARNING_CONFIG_FAILED;
-
+#if 0 /* FIXME Feature not yet implemented */
```

Comment:
I personally prefer that we just delete code instead of putting #if 0 's around it.  @louberger you are the one who is most interested in this code.  Opinions?

Reply:
@paulzlabn opinions?

Reply:
I don't see a need to keep this code block around - after all these years it seems unlikely we will need it. I'm OK with its deletion.

Reply:
then let's remove the code instead of #if 0'ing it